libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -1830,28 +1830,28 @@ def produce_tests():
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-//
-// WARNING: This test was generated by {script_name}
+
+// WARNING: This test was generated by
+// {script_name}
```

Comment:
This is not over-my-dead-body obviously, but I think the output was better with this on the same line. It looks awkward when on a different line.